### PR TITLE
Possibly fix disabled type to be bool

### PR DESCRIPTION
**Pull request by johnmcmahon**

### CODE REVIEW

File: workflow/common.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ type Trigger struct {
 	PercolationId piazza.Ident `json:"percolationId"`
 	CreatedBy     string       `json:"createdBy"`
 	CreatedOn     time.Time    `json:"createdOn"`
-	Disabled      byte         `json:"disabled"` // TODO: this will panic() if using a bool instead of a byte???
+	Disabled      bool         `json:"disabled"` // TODO: this will panic() if using a bool instead of a byte???
```
Comment (johnmcmahon) [inline workflow/common.go:118]: I guess if this works i can take out this comment

File: workflow/WorkflowService.go [hunk 2]
```diff
@@ -478,7 +478,7 @@ func (service *WorkflowService) PostEvent(event *Event) *piazza.JsonResponse {
 					results[triggerID] = statusNotFound(triggerID)
 					return
 				}
-				if trigger.Disabled == 1 {
+				if trigger.Disabled == true {
 					//results[triggerID] = statusOK(triggerID)
```
Comment (johnmcmahon) [inline workflow/WorkflowService.go:482]: does this need to be uncommented?

File: workflow/common.go (hunk 1 above)
Comment (mpgerlek) [inline workflow/common.go:118]: yup

File: workflow/WorkflowService.go (hunk 2 above)
Comment (mpgerlek) [inline workflow/WorkflowService.go:482]: No -- remove that whole line, it is (unrelated) dead code

**Pull request merged**